### PR TITLE
651 geoserver role

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: roles/pulibrary.tomcat8/molecule/default/tests/test_tomcat8.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+def test_for_tomcat8(host, name):
+    pkg = host.package(name)
+
+    assert pkg.is_installed
+
+
+def test_tomcat8_listening_http(host):
+    socket = host.socket('tcp://0.0.0.0:8080')
+
+    assert socket.is_listening
```
Comment (jrgriffiniii) [inline roles/pulibrary.tomcat8/molecule/default/tests/test_tomcat8.py]: 👍

File: roles/pulibrary.geoserver/molecule/default/tests/test_default.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_geoserver_war_file(host):
+    f = host.file('/var/lib/tomcat8/webapps/geoserver.war')
+
+    assert f.exists
```
Comment (jrgriffiniii) [inline roles/pulibrary.geoserver/molecule/default/tests/test_default.py]: 👍

File: roles/pulibrary.tomcat8/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+passed in as parameters) is always nice for users too:
+
+    - hosts: servers
+      roles:
+         - { role: roles/pulibrary.tomcat8 }
+
+License
+-------
+
+BSD
```
Comment (jrgriffiniii) [inline roles/pulibrary.tomcat8/README.md]: Is this in conflict with https://github.com/pulibrary/princeton_ansible/pull/660/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#diff-df6702ec8636d6df70580f769dfd2d6eR6 and https://github.com/pulibrary/princeton_ansible/pull/660/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#diff-80361ed3024805b172f29ab368897fb5R6?

Comment (kayiwa) [reply roles/pulibrary.tomcat8/README.md]: hmmm. the former is correctly in the Tomcat8 Role (which the latter Geoserver one) depends on. It's late in the day and my eyes may be tricking me.

**Pull request merged**